app/services/jbpm_process_service.rb

```diff
@@ -0,0 +1,37 @@
+class JbpmProcessService
+  attr_accessor :request
+
+  def initialize(request)
+    self.request = request
+  end
+
+  def start
+    options = template.process_setting
+    Kie::Service.call(KieClient::ProcessInstancesBPMApi, options) do |bpm|
+      bpm.start_process(options['container'], options['process'], :body => process_options)
```

Comment:
For clarification, it's better to use the same parameter name as defined in Kie server doc, like
`bpm.start_process(options['container_id'], options['process_id'], :body => process_options)`